### ISSUE TITLE
Can't enable blob caching on a web application and his extend

### ISSUE THREAD

**Issue by wickedave** [PowerShell/SharePointDsc#636]
**Details of the scenario you try and problem that is occurring:**
I can't enable the blob caching on a web application and his extend. It is telling me that a conflict was detected between resources. Is it possible to make sure blob caching can be enable on the same webapplication twice?

**The DSC configuration that is using the resource:**
```
Configuration Example 
{
    param(
        [Parameter(Mandatory = $true)]
        [PSCredential]
        $SetupAccount
    )
    Import-DscResource -ModuleName SharePointDsc

    node localhost {
        SPWebApplication HostNameSiteCollectionWebApp
        {
            Name                   = "SharePoint Sites"
            ApplicationPool        = "SharePoint Sites"
            ApplicationPoolAccount = "CONTOSO\svcSPWebApp"
            AllowAnonymous         = $false
            AuthenticationMethod   = "NTLM"
            DatabaseName           = "SP_Content_01"
            DatabaseServer         = "SQL.contoso.local\SQLINSTANCE"
            Url                    = "http://example.contoso.local"
            Port                   = 80
            Ensure                 = "Present"
            PsDscRunAsCredential   = $SetupAccount
        }
		
		SPBlobCacheSettings BlobCacheSettings 
        {
            WebAppUrl = "http://example.contoso.local"
            Zone = "Default"
            EnableCache = $true
            Location = "F:\BlobCache"
            MaxSizeInGB = 10
            FileTypes = "\.(gif|jpg|png|css|js)$"
            PsDscRunAsCredential = $SetupAccount
        }
		
		SPWebApplicationExtension IntranetZone
        {
            WebAppUrl              = "http://example.contoso.local"
            Name                   = "Contoso Intranet Zone"
            AllowAnonymous         = $false
            AuthenticationMethod   = "NTLM"
            Url                    = "http://intranet.contoso.local"
            Zone                   = "Intranet"
            HostHeader             = "intranet.contoso.local"
            Path                   = "c:\inetpub\wwwroot\wss\VirtualDirectories\intranet"
            UseSSL                 = $false
            Port                   = 80
            Ensure                 = "Present"
            PsDscRunAsCredential   = $SetupAccount
        }
		
		SPBlobCacheSettings BlobCacheSettingsIntranet
        {
            WebAppUrl = "http://example.contoso.local"
            Zone = "Intranet"
            EnableCache = $true
            Location = "F:\BlobCache-Intranet"
            MaxSizeInGB = 10
            FileTypes = "\.(gif|jpg|png|css|js)$"
            PsDscRunAsCredential = $SetupAccount
        }
    }
}

```

**Version of the Operating System and PowerShell the DSC Target Node is running:**
Windows Server 2016 with SharePoint 2016

**Version of the DSC module you're using:**
1.7.0.0

**Comment by wickedave** [comment]
Here is more details:
Resources have identical key properties but there are differences in the following non-key properties: 'Location;Zone'

**Comment by ykuijs** [comment]
@wickedave This is a duplicate with #609, which is fixed in v1.8.

Please test with v1.8 and reopen the issue if you still encounter the issue. Closing issue for now.
